bundle/manifests/cert-manager-operator.clusterserviceversion.yaml

```diff
@@ -664,7 +664,10 @@ spec:
                 image: openshift.io/cert-manager-operator:latest
                 imagePullPolicy: Always
                 name: cert-manager-operator
-                resources: {}
+                resources:
```

Comment:
Is the operator container resource requests/limits customizable in any way?
We're setting it to these values here in the CSV which is a default. Asking this because, the original bug report is also about the operator resources and not just for operand.

Reply:
We are only setting the `requests` so should be ok, but yes limits need to be customizable. Can be done using `subscription.spec.config` so not an issue.

Reply:
Ok, we need to document it later (like we did for logLevels).
+doc-ack, cc: @xenolinux